### DISCUSSION TITLE
Qn: Does usebasejump support multi tenancy?

### DISCUSSION THREAD

**Discussion by BrightonMboya** [usebasejump/basejump#90] [General]
I have found a case where, 
1. Create an account (as in an org owner and then add team to your account), now personA can belong to Organization A and its account is created on both basejump.accounts and auth.users schema
2. now personA can also belong to organization B, and when personA receives the invitation to organization B he cant create an account because his account already exists from organization A.

I dug and found out the email column on auth.user schema is unique and technically you cant support multitenancy architecture unless there's a way to manually edit the column, but who knows what will cause down the line.

Am I missing sth or this is what is currently supported?

**Comment by tejesh002** [comment]
I too facing the same issue. Since supabase have auth schema unique constrain.

**Reply by BrightonMboya** [reply]
might be a skill issue on my end but I ended up switching to auth.js and added more additional data schema to support multi tennacy. You can check this file and see if you can get the sense of how to achieve the same thing on ur project
https://github.com/BrightonMboya/kilimo/blob/main/prisma/schema.prisma
